webactivity.py

```diff
@@ -522,19 +524,28 @@ def _key_press_cb(self, widget, event):
             elif key_name == 'Right':
                 _logger.debug('keyboard: Go forward')
                 browser.go_forward()
-            elif key_name == 'r':
+            elif key_name in ['r', 'F5']:
                 _logger.debug('keyboard: Reload')
                 browser.reload()
             elif Gdk.keyval_name(event.keyval) == "t":
                 self._tabbed_view.add_tab()
-            elif key_name == 'w':
+            elif key_name in ['w', 'F4']:
                 _logger.debug('keyboard: close tab')
                 self._tabbed_view.close_tab()
+            elif key_name ==  'F11':
+                if self.count % 2 == 0 :   
+                    self._primary_toolbar.hide()
+                    self.count = self.count + 1
+                else:
+                    self._primary_toolbar.show_all()
+                    self.count = self.count + 1
+                
             else:
                 return False
 
             return True
 
+
```

Comment:
:angry: No more lines plz!  We don't want to waste bytes!  (Also it isn't with the style)